src/types.js

```diff
@@ -234,8 +234,14 @@ export type MessageLike =
       ...Outbox,
     }>;
 
-// From docs: https://formatjs.io/docs/react-intl/api/#formatmessage
-type IntlMessageFormatValue = string | number | boolean | null | void;
+// We adapt `IntlMessageFormatValue` from the `react-intl` libdef,
+// which doesn't quite match the doc at
+// https://formatjs.io/docs/react-intl/api/#formatmessage.
+type PrimitiveType = string | number | boolean | null | void | Date;
```

Comment:
Ah, right. I copied the name from the libdef; should have adapted it accordingly. Looks like, below, you've pointed out a way to avoid having to copy all this stuff in anyway, though, nice.